paper/paper.md

```diff
@@ -11,36 +11,57 @@ authors:
+Understanding the impact of extreme events on people's ability to access critical service is key to designing appropriate programs to minimize the overall impact. In the event of a disaster, damage to the electric system and related infrastructure (e.g. downed power lines, flooded equipment, hacked communication systems, damaged roads,  etc.) can impact people's access to critical services including not just electricity but also shelter, food, healthcare, and more.  Moreover, the individual and community ease of service access--both before and during such events--may be correlated with household income and other demographics, with some historically disadvantaged communities potentially experiencing more difficult access to critical services. There is a key need to better understanding such impacts and evaluate options to improve resilience and equity. The Equity and Resilience Analysis for electric Distribution systems (ERAD) tool is a free and open-source software package designed to be used by researchers, utilities, local governments, community groups, and other stakeholders to fill this need.
 
-ERAD is a free, open-source Python toolkit for computing equity and resilience measures in the face of hazards like earthquakes and flooding. It uses graph database to store data and perform computation at the household level for a variety of critical services that are connected by power distribution network. It uses asset fragility curves, which are functions that relate hazard severity to survival probability for power system assets including cables, transformers, substations, roof-mounted solar panels, etc. recommended in top literature. Programs like undergrounding, microgrid, and electricity backup units for critical infrastructures may all be evaluated using metrics and compared across different neighborhoods to assess their effects on equity and resilience.
+# Statement of need
 
-ERAD is designed to be used by researchers, students, community stakeholders, distribution utilities to understand and possibly evaluate effectiveness of different post disaster programs to improve resilience and equity. It was funded by National Renewable Energy Laboratory (NREL) and made publicy available with open license.
+~~In the last few years, we have seen a surge in natural disaster events affecting the power grid, in part due to increased frequency and intensity of extreme weather due to climate change. Similarly, human-driven malicious acts, such as cybersecurity attacks can have similar, wide-spread impacts on the grid. Grid outages in turn effect other services, which have a direct impact on people's lives during an emergency.~~
```

Comment:
Propose deleting this part. The Summary already includes a similar motivation and the statement of need seems to be more about why a tool is needed rather than the overall problem motivation